_slides/03-AmandaBrockUK.md

```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+Cool for _Cats_
 
 ![catstello](https://octodex.github.com/images/catstello.png)
```

Comment:
This is a great image, but if you wanted to change it, go [here](https://octodex.github.com/)